src/App.css

```diff
@@ -1,18 +1,124 @@
+:root {
+  --primary-color: #1a73e8;
+  --hover-primary: #1557b0;
+  --background-color: #f8f9fa;
+  --column-background: #ffffff;
+  --border-color: #dadce0;
+  --text-primary: #202124;
+  --text-secondary: #5f6368;
+}
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
:root {
  --primary-color: #1a73e8;
  --hover-primary: #1557b0;
  --background-color: #f8f9fa;
  --column-background: #ffffff;
}

@media (prefers-color-scheme: dark) {
  :root {
    --primary-color: #8ab4f8;
    --hover-primary: #aecbfa;
    --background-color: #202124;
    --column-background: #303134;
    --text-primary: #ffffff;
    --text-secondary: #9aa0a6;
  }
}
```
Consider adding a prefers-color-scheme media query to support dark mode using the CSS variables already defined. This would improve accessibility and user experience in different lighting conditions.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;